buildlib/pr/codestyle.yml

```diff
@@ -63,6 +63,33 @@ jobs:
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
 
+  - job: author
+    displayName: AUTHORS file check
```

Comment:
maybe include the word `update` in the name?

Reply:
fixed